scripts/terraform-docs.sh

```
@@ -1,9 +1,26 @@
 #!/bin/bash
 
 set -e
+# renovate: datasource=github-releases depName=terraform-docs/terraform-docs
+TERRAFORM_DOCS_VERSION=v0.22.0
+BINARY=terraform-docs
+set +e
+INSTALLED_TERRADOCS_VERSION="$(terraform-docs --version | head -1 | cut -d' ' -f3)"
+set -e
+if [[ "$TERRAFORM_DOCS_VERSION" != "$INSTALLED_TERRADOCS_VERSION" ]]; then
+  curl -sSLo ./terraform-docs.tar.gz https://terraform-docs.io/dl/${TERRAFORM_DOCS_VERSION}/terraform-docs-${TERRAFORM_DOCS_VERSION}-$(uname)-amd64.tar.gz
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)